server/src/main/java/org/candlepin/resource/dto/ActivationKeyData.java

```diff
@@ -0,0 +1,757 @@
+/**
+ * Copyright (c) 2009 - 2012 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.resource.dto;
+
+import org.candlepin.model.activationkeys.ActivationKey;
+import org.candlepin.model.activationkeys.ActivationKeyPool;
+import org.candlepin.model.ContentOverride;
+import org.candlepin.model.Owner;
+import org.candlepin.model.Pool;
+import org.candlepin.model.Product;
+import org.candlepin.model.Release;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+
+
+/**
+ * DTO representing the activation key data exposed to the API layer.
+ */
+@XmlRootElement
+public class ActivationKeyData {
+
+    /**
+     * DTO class representing activation key pool data.
+     * <p/>
```

Comment:
I think that in javadoc you should not use <p/> but you should use 'broken' markup <p> see [1]. It seems that in other parts of Candlepin javadoc, we tend to use broken markup as well.

[1] http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format